packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -676,9 +681,36 @@ export default class FocusZone extends React.Component<FocusZoneProps> implement
 
   /**
    * Walk up the dom try to find a focusable element.
-   * TODO
    */
-  tryInvokeClickForFocusable(): boolean {
+  tryInvokeClickForFocusable(targetElement: HTMLElement): boolean {
```

Comment:
in v0, we are handling enter or space in the behaviors, and they are in some cases different based on the component type (for example in selectable tree, space select the item, enter executes click on it). If we activate this, we would need to do changes across components to prevent double clicking I think

Reply:
but that would be confusing - since the components are not always inside of focus zone, we would need to handle this differently in different scenarios. How does it work in v7?

Reply:
The FocusZone can technically be wrapped around any focusable element, including `div` elements with the `data-is-focusable` attribute. When this attribute is present, the FocusZone makes the element focusable. We've tried to follow a principle of ensuring elements we make focusable are interactive (except for special cases, like "parking" scenarios.) Otherwise it feels like an accessibility concern without the implementer ensuring they do the right thing (I wouldn't argue that they *should* ensure their components are accessible, but we can't control that.)

I'm believe we do not hit the double click event scenario, because when we raiseClick, we return true, stopping propagation and canceling the normal behavior (which may have been simulate the click.)

I'll also say, it seems weird that the FocusZone would effectively "patch" the element's keypress behavior. It seems like the interaction model should be up to the component itself, and whether its inside or outside of the FocusZone shouldn't really matter.

If we remove this behavior I believe some user cases would be affected. If we keep this behavior, I believe v0 scenarios are ok. I'd recommend we keep it as an opt in defaulted to false. In v7 code, we could default it to true to avoid dispruption, and in v8 default it to false with a clear release note on the change.

Reply:
agree - we have similar case, where we have different default for v0/v7 already: https://github.com/microsoft/fluentui/blob/d8c9366bfd9ea973eaf71ae9bcb7a4fe6e43b945/packages/fluentui/react-bindings/src/accessibility/getAccessibility.ts#L36